src/index.html

```diff
@@ -2,13 +2,60 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img src="./images/logo.png" alt="Moyo logo" class="logo">
+      <nav class="navigation">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">
+              APPLE
```

Comment:
```suggestion
              apple
```
Варто писати в нижньому регістрі, а в css вже задавати `text-transform: uppercase`

Reply:
чому?

Reply:
Якщо потім хтось захоче додати ще один розділ до навігації, то йому не доведеться задумуватися у якому фотматі це писати і якби хтось не написав відображатиметься усе коректно